examples/datatable.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Datatable demo feature
+
+  I want to find TestCafe repository by Google search
+
+  @googleHook
+  Scenario: Searching for Datatable by Google
```

Comment:
I think this is a bad example. I would rather use the [TestCafé example page](https://devexpress.github.io/testcafe/example/) and check some different checkboxes that are defined in a data table.

Reply:
agree attached new example